bin/cherry-pick.mjs

```diff
@@ -0,0 +1,416 @@
+/**
+ * External dependencies
+ */
+import fetch from 'node-fetch';
+import readline from 'readline';
+
+import { spawnSync } from 'node:child_process';
```

Comment:
Isn't it Node 18 specific API `node:`? 😄 

I'm sure it won't work with Node 14.

Reply:
It doesn't work with Node 12 for me, but it did work with Node 14:

```
; nvm use 14
Now using node v14.19.0 (npm v6.14.16)

; node bin/cherry-pick.mjs
You are on branch "automate/cherry-picking-for-a-release".
This script will:
...
```

Reply:
It looks like they added this new syntax to older versions of Node.js, too:

<img width="496" alt="Screenshot 2022-06-28 at 12 23 22" src="https://user-images.githubusercontent.com/699132/176156591-86b903a8-bd00-4484-91c6-83fe2a19bcd8.png">

It should be good for this type of tool 👍🏻